wallet/src/main/scala/org/bitcoins/wallet/internal/UtxoHandling.scala

```diff
@@ -379,7 +417,10 @@ private[wallet] trait UtxoHandling extends WalletLogger {
     for {
       infos <- spendingInfoDAO.findAllPendingConfirmation
       _ = logger.debug(s"Updating states of ${infos.size} pending utxos...")
-      updatedInfos <- updateUtxoConfirmedStates(infos)
-    } yield updatedInfos
+      receivedUtxos = infos.filter(_.state.isInstanceOf[ReceivedState])
```

Comment:
What to do about `TxoState.DoesNotExist`? It is not a `ReceivedState` or `SpentState`, so it gets filtered out here.

Reply:
We honestly could remove `DoesNotExist`, I don't think we have a use for it